### PR TITLE
Redirection localization

### PR DESCRIPTION
**Pull request by Fuzznimp**
## Description

- Increased header width.
- Fixed `NEXT_LOCALE` implementation.
- Localized all the missing breadcrumbs.
- Localized profile pages.

## Type of change

- [ ] **Chore** (non-breaking change which refactors / improves the existing code base).
- [x] **Bug fix** (non-breaking change which fixes an issue).
- [x] **New feature** (non-breaking change which adds functionality).
- [ ] **Breaking change** (fix or feature that would cause existing functionality to not work as expected).

## Checklist

- [ ] My code follows the style [**contributing guidelines**][contributing_file] of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.

[contributing_file]: https://github.com/fewlinesco/connect-account/blob/master/README.adoc

### CODE REVIEW

**Comment by frieda-ewlines** [comment]
Task linked: [CU-9kypmk [fix] - Localized redirection after callback + showmore btn](https://app.clickup.com/t/9kypmk)

File: src/configs/intl.ts [hunk 1]
```diff
@@ -61,7 +61,27 @@ function formatNavigation(locale: string, id: string): string {
 }
 
 function getLocaleFromRequest(request: NextApiRequest, span: Span): string {
-  let locale = request.cookies["NEXT_LOCALE"] || "en";
+  if (!request.headers["accept-language"]) {
+    let locale = request.cookies["NEXT_LOCALE"] || "en";
+
+    if (locale[0] === '"') {
+      locale = locale.slice(1, -1);
+    }
```
Comment (Oluane) [inline src/configs/intl.ts:69]: I think you could get rid of that now 🙂

Comment (Fuzznimp) [reply src/configs/intl.ts:69]: Nice catch :D.

**Review by Oluane — APPROVED** [review]

**Comment by Oluane** [comment]
image


We seem to have an issue here 😆

**Review by Oluane — APPROVED** [review]

**Pull request merged**